libraries/ms-helper/inc/exported_enums.h

```diff
@@ -121,6 +121,16 @@ typedef enum {
   NUM_EE_EBRAKE_STATES,
 } EEEbrakeState;
 
+typedef enum {
```

Comment:
remove this, the rest of the car doesn't need to know about what your inputs are.